pkg/process/checks/net.go

```diff
@@ -144,7 +144,15 @@ func batchConnections(cfg *config.AgentConfig, groupID int32, cxs []*model.Conne
 
 	for len(cxs) > 0 {
 		batchSize := min(cfg.MaxConnsPerMessage, len(cxs))
-		ctrIDForPID := Process.filterCtrIDsByPIDs(connectionPIDs(cxs[:batchSize]))
+		// get the container and process relationship from either process check or container check
```

Comment:
Can we extract this into it's own function to keep the code clean?

Reply:
Sure, I'll do that.

Reply:
```
ctrIDForPID := filterCtrIDsByPID(connectionPIDs(cxs[:batchSize]))
...

func filterCtrIDsByPID([]int32 pids) map[int32]string {
		if Process.lastRun.IsZero() { // Process check never ran, so use container check instead.
			return Container.filterCtrIDsByPIDs(connectionPIDs(cxs[:batchSize]))
		} 
		return Process.filterCtrIDsByPIDs(connectionPIDs(cxs[:batchSize]))
}